web/viewer.css

```diff
@@ -1264,6 +1223,47 @@ dialog :link {
 }
 
 #secondaryToolbar {
+  .toolbarButton {
+    border-radius: 0;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    padding: 0 0 1px;
+    padding-inline-start: 36px;
+    position: relative;
+    min-height: 26px;
+    min-width: 100%;
+    text-align: start;
+    white-space: normal;
+    width: auto;
+
+    &::before {
+      inset-inline-start: 12px;
+      opacity: var(--doorhanger-icon-opacity);
+      top: 5px;
+    }
+
+    &:not(.toggled):is(:hover, :focus-visible) {
+      background-color: var(--doorhanger-hover-bg-color);
+      color: var(--doorhanger-hover-color);
+    }
+
+    > span {
+      display: unset;
+      padding-inline-end: 4px;
+    }
+  }
+
+  a.toolbarButton {
```

Comment:
I would like that, but it'd basically require a way to, in nested CSS, detect on which element the class is being applied. I don't think there is way to do that (or at least I'm not aware of it).

Reply:
It might be abusing notation a tiny bit, but the following does seem to work:
```diff
diff --git a/web/viewer.css b/web/viewer.css
index 7b1236e10..5da060f81 100644
--- a/web/viewer.css
+++ b/web/viewer.css
@@ -1252,15 +1252,15 @@ dialog :link {
       display: unset;
       padding-inline-end: 4px;
     }
-  }

-  a.toolbarButton {
-    padding-top: 5px;
-    text-decoration: none;
+    &:is(a) {
+      padding-top: 5px;
+      text-decoration: none;

-    &[href="#"] {
-      opacity: 0.5;
-      pointer-events: none;
+      &[href="#"] {
+        opacity: 0.5;
+        pointer-events: none;
+      }
     }
   }
```

Reply:
Thanks; I'll give that a try and prepare a follow-up for this and another thing I noticed along the way.